bot/exts/utils/subscription.py

```diff
@@ -0,0 +1,192 @@
+import discord
+from bot.bot import Bot
+from bot.constants import Colours, Emojis, Roles
+from discord import Embed
+from discord.ext import commands
+from discord.ext.commands import Cog, Context
+
+
+class Subscription(Cog):
+    """Subscribing and Unsubscrbing to announcements and polls notifications."""
+
+    def __init__(self, bot: Bot) -> None:
+        self.bot = bot
+
+    async def apply_role(self, ctx: Context, role_name: discord.Role) -> bool:
+        """Adding Roles and returning True otherwise False."""
```

Comment:
```suggestion
        """Adding roles and returning `True` otherwise `False`."""
```
You are pointing towards some code of python, or keywords so these always be enclosed with back ticks. Apply this change at the other required places too.

Another point, this docstring doesn’t seem to explain anything what the function does, we can understand it returns True/False by the return type annotation but what cases are they returned?